domain/timeline/src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/domain/timeline/Timeline.kt

```diff
@@ -10,7 +10,7 @@ class Timeline(
 ) {
 
   val events: MutableList<TimelineEvent>
-    get() = field.also { events -> events.sortByDescending { it.eventDateTime } }
+    get() = field.also { events -> events.sortBy { it.timestamp } }
```

Comment:
The Timeline events should be ordered in ascending chronological order.

Reply:
I agree with that from a domain / data point of view 👍 

I think the prototype screen however is wrong/misleading in this respect. I suspect once we develop the UI (post MVP) , if we have to do it as per the current prototype, they will soon change their mind 😁 because (in its current form of earliest event first) it means the CIAG will likely have to scroll to the bottom of the page to find the stuff they are really interested in (ie. I think they'll find the most recent goals/events more useful than the earlier stuff)

Clearly this is a UI concern though (how they want it rendered). So I agree with your change here that from a domain/data POV they should be ordered in ascending order; but I suspect either the UI or the controller code will re-sort the events based on how the UI wants to render them